src/components/MultiLink/MultiLink.vue

```diff
@@ -36,7 +36,11 @@ export default Vue.extend({
     simulationLinks(): SimulationLink[] | null {
       if (this.network !== null) {
         return this.network.edges.map((link: Link) => {
-          const newLink: SimulationLink = { ...link, source: link._from, target: link._to };
+          const newLink: SimulationLink = {
+            ...JSON.parse(JSON.stringify(link)),
+            source: link._from,
+            target: link._to,
+          };
```

Comment:
I agree that going through JSON and back comprehensively deep-copies an object, but it's very slow. I'd open an issue to track any performance problems that might result from this.

Reply:
Opened #138